### PR TITLE
Drop 0.4 support

### PR DESCRIPTION
**Pull request by musm**
Needs a squash

### CODE REVIEW

File: deps/build.jl [hunk 1]
```diff
@@ -21,7 +20,7 @@ if !isfile(target) || !isfile(tagfile) || readchomp(tagfile) != "$vers $(Sys.WOR
         run(unpack_cmd(tarball, ".", ".gz", ".tar"))
         cd(srcdir) do
             println("Compiling libblosc...")
-            run(`$make -f ../../make.blosc LIB=../../$target`)
+            run(`$MAKE_CMD -f ../../make.blosc LIB=../../$target`)
```
Comment (musm) [inline deps/build.jl]: Note ver 0.6 of BinDeps takes care of  the `gmake` vs `make`  command for `bsd` and non-`bsd` setups

**Comment by musm** [comment]
Ideally we could drop 0.5, but perhaps that should wait till end of summer?

**Comment by stevengj** [comment]
LGTM.

**Comment by stevengj** [comment]
I would prefer not to consider dropping 0.5 until 0.7 is a bit further along.

**Comment by musm** [comment]
Thanks, perhaps a tag ?

**Comment by stevengj** [comment]
v0.3.0 was already tagged

**Pull request merged**